purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterEventType.kt

```diff
@@ -0,0 +1,23 @@
+package com.revenuecat.purchases.customercenter.events
+
+/**
+ * Types of events that can be tracked in the Customer Center. Meant for RevenueCatUI use.
+ */
+enum class CustomerCenterEventType(val value: String) {
```

Comment:
Should this also be annotated with the `@ExperimentalPreviewRevenueCatPurchasesAPI` annotation?